sites/public/pages/applications/preferences/displaced.tsx

```diff
@@ -0,0 +1,307 @@
+  >({
+    defaultValues: {
+      general:
+        displacedPreference?.options?.find((options) => options.key == "general").checked === true,
+      missionCorridor:
+        displacedPreference?.options?.find((options) => options.key == "missionCorridor")
+          .checked === true,
+      none: displacedPreference?.claimed === false,
+      displacedName: displacedPreference?.options?.find((options) => options.key == "general")
+        ?.extraData[0]?.value,
+      displacedAddress: displacedPreference?.options?.find((options) => options.key == "general")
+        ?.extraData[1]?.value,
+    },
+    shouldFocusError: false,
+  })
+  const onSubmit = (data) => {
+    if (!data.none) {
+      conductor.completeSection(4)
+    }
+
+    conductor.currentStep.save({ ...data })
+    conductor.routeToNextOrReturnUrl()
+  }
+
+  const generalOptionSelected = watch("general")
+
+  return (
+    <FormsLayout>
+      <FormCard header={listing?.name}>
+        <ProgressNav
+          currentPageSection={currentPageSection}
+          completedSections={application.completedSections}
+          labels={conductor.config.sections.map((label) => t(`t.${label}`))}
+        />
+      </FormCard>
+
+      <FormCard>
+        <FormBackLink url={conductor.determinePreviousUrl()} />
+
+        <div className="form-card__lead border-b">
+          <h2 className="form-card__title is-borderless">{t("application.preferences.title")}</h2>
+
+          <p className="field-note mt-5">{t("application.preferences.preamble")}</p>
+
+          <p className="field-note mt-5 text-center">
+            {t("t.pageXofY", { num: 2, total: conductor.preferenceStepsTotal })}
+          </p>
+        </div>
+
+        {Object.entries(errors).length > 0 && (
+          <AlertBox type="alert" inverted closeable>
+            {t("errors.errorsToResolve")}
+          </AlertBox>
+        )}
+
+        <Form onSubmit={handleSubmit(onSubmit)}>
+          <div className="form-card__group px-0 pb-3">
+            <p className="field-note">{t("application.preferences.selectBelow")}</p>
+          </div>
+
+          {preferenceOptions.map((option) => (
+            <div key={option} className="form-card__group px-0 border-b">
+              <div className={"field " + (errors.none ? "error" : "")}>
+                <input
+                  type="checkbox"
+                  id={option}
+                  name={option}
+                  ref={register}
+                  onChange={() => {
+                    setTimeout(() => {
+                      setValue("none", false)
+                      void trigger("none")
+                    }, 1)
+                  }}
+                />
+                <label htmlFor={option} className="font-semibold uppercase tracking-wider">
+                  {t(`application.preferences.displacedTenant.${option}.label`)}
+                </label>
+              </div>
+
+              <p className="ml-8 -mt-3">
+                <button
```

Comment:
I think will be good to create a separate component for this or add a variant to the existing component (we have the same button in 2 steps).